app/ui.js

```diff
@@ -1714,6 +1701,21 @@ const UI = {
  */
 };
 
-export default UI;
+// Set up translations
+var LINGUAS = ["de", "el", "nl", "sv"];
+l10n.setup(LINGUAS);
+if (l10n.language !== "en" && l10n.dictionary === undefined) {
```

Comment:
Did you consider my approach of hiding the loading logic in the localisation code?